var/build-config.json

```diff
@@ -1,9 +1,25 @@
 {
-  "boardingLocationTags": ["gtfs:stop_id", "ref"],
+  "boardingLocationTags": [
+    "gtfs:stop_id",
+    "ref"
+  ],
+  "dataImportReport": true,
   "embedRouterConfig": false,
   "osmDefaults": {
     "timeZone": "America/New_York"
   },
+  "transitFeeds": [
+    {
+      "type": "gtfs",
+      "feedId": "massport-ma-us",
+      "source": "https://data.trilliumtransit.com/gtfs/massport-ma-us/massport-ma-us.zip"
+    },
+    {
+      "type": "gtfs",
+      "feedId": "mbta-ma-us",
+      "source": "mbta-ma-us.gtfs.zip"
```

Comment:
Are we only not using the URL for our feed because we have the env var for it? It'd be nice if we could be consistent.

Reply:
Yeah, I thought referencing environment variables wouldn't work here but it turns out it works! I've since created consistency by defining all the external resources in this config! ✨